cupy/cuda/jitify.pyx

```diff
@@ -69,12 +73,15 @@ cpdef _add_sources(dict sources):
 
 @atexit.register
```

Comment:
Now that the cache is immutable, how about changing the timing of persisting cache from `atexit` to the time the cache is generated? `atexit` is often unstable (e.g., user hitting Ctrl-C multiple times) and I'd like avoid using it whenever possible.

Reply:
Yes, now that the cache is immutable, we shouldn't need any `atexit` work anymore, good catch!

Reply:
Fixed in 9420ec1.